spec/services/responsibility_service_spec.rb

```diff
@@ -1,166 +1,332 @@
+            o.sentence.release_date = release_date
+          }
+        }
+
         context 'when offender has less than 12 weeks to serve' do
-          scenario 'is supporting' do
-            resp = described_class.calculate_pom_responsibility(offender_welsh_crc_lt_12_wk)
+          let(:release_date) { DateTime.now.utc.to_date + 2.weeks }
 
-            expect(resp).to eq 'Supporting'
+          it 'is supporting' do
+            resp = described_class.calculate_pom_responsibility(offender)
+
+            expect(resp).to eq ResponsibilityService::SUPPORTING
           end
         end
 
         context 'when offender has more than twelve weeks to serve' do
-          scenario 'is responsible' do
-            resp = described_class.calculate_pom_responsibility(offender_welsh_crc_gt_12_wk)
+          let(:release_date) { DateTime.now.utc.to_date + 13.weeks }
+
+          it 'is responsible' do
+            resp = described_class.calculate_pom_responsibility(offender)
 
-            expect(resp).to eq 'Responsible'
+            expect(resp).to eq ResponsibilityService::RESPONSIBLE
           end
         end
       end
 
       context 'when NPS case' do
         context 'when new case (sentence date after February 4 2019)' do
+          let(:offender_welsh_nps_new_case_gt_10_mths) {
+            Nomis::Offender.new.tap { |o|
+              o.welsh_offender = true
+              o.case_allocation = 'NPS'
+              o.sentence = Nomis::SentenceDetail.new
+              o.sentence.sentence_start_date = DateTime.new(2019, 2, 19).utc
+              o.sentence.release_date = DateTime.now.utc.to_date + 11.months
+            }
+          }
+
+          let(:offender_welsh_nps_new_case_lt_10_mths) {
+            Nomis::Offender.new.tap { |o|
+              o.welsh_offender = true
+              o.case_allocation = 'NPS'
+              o.sentence = Nomis::SentenceDetail.new
+              o.sentence.sentence_start_date = DateTime.new(2019, 2, 20).utc
+              o.sentence.release_date = DateTime.now.utc.to_date + 9.months
+            }
+          }
+
           context 'when time left to serve is greater than 10 months' do
-            scenario 'is responsible' do
+            it 'is responsible' do
               resp = described_class.calculate_pom_responsibility(offender_welsh_nps_new_case_gt_10_mths)
 
-              expect(resp).to eq 'Responsible'
+              expect(resp).to eq ResponsibilityService::RESPONSIBLE
             end
           end
 
           context 'when time left to serve is less than 10 months' do
-            scenario 'is supporting' do
+            it 'is supporting' do
               resp = described_class.calculate_pom_responsibility(offender_welsh_nps_new_case_lt_10_mths)
 
-              expect(resp).to eq 'Supporting'
+              expect(resp).to eq ResponsibilityService::SUPPORTING
             end
           end
         end
 
         context 'when old case (sentence date before February 4 2019)' do
```

Comment:
Do we want a 'if new case' and if 'old case' test for English offenders as well?

Reply:
There was - I've updated the comments to make this more explicit